src/main/java/mate/controller/LoginController.java

```diff
@@ -0,0 +1,43 @@
+package mate.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+import mate.exception.AuthenticationException;
+import mate.lib.Injector;
+import mate.model.Driver;
+import mate.service.AuthenticationService;
+
+@WebServlet(urlPatterns = "/login")
```

Comment:
I see that you configured most of the URL patterns in web.xml, so why do you use in this task annotations for URL patterns. It's not consistent, choose one option

Reply:
ok, I prefer annotations